vizro-core/src/vizro/models/_navigation/_accordion.py

```diff
@@ -77,23 +77,24 @@ def _create_default_accordion(self):
                     class_name="accordion",
                     persistence=True,
                     persistence_type="session",
+                    always_open=True,
```

Comment:
I definitely prefer to have this setting, but it deviates a bit from the behaviour we've discussed with J. Let's just quickly align with him in our catch-up